mo_net/samples/cbow.py

```
@@ -469,54 +537,44 @@ def infer(model_path: Path):
-
-    while True:
-        try:
-            user_input = input("> ").strip()
-            if user_input.lower() in ["quit", "exit", "q"]:
-                break
-
-            words = user_input.split()
-            if len(words) != 2:
-                click.echo("Please enter exactly two words separated by space")
-                continue
+    cbow_model = CBOWModel.load(open(model_path, "rb"), training=False)
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)